test/files/neg/erroneous-anon-classes.scala

```diff
@@ -0,0 +1,90 @@
+import scala.language.higherKinds
```

Comment:
IMHO, it would be better to follow the convention so far, and split this test file into different files, specially since each bug test case uses a slightly different set of features of Scala: 
- Bug 11337 involves type parameters, but of the simple kind, and type members. 
- Bug 10015 uses refinement types  `x : { def g }`, but no type parameters of any kind, and no type members.
Etc. 

It may well happen that some change could break one of these, but not the others.